src/picongpu/include/plugins/kernel/CopySpecies.kernel

```diff
@@ -71,6 +71,7 @@ struct ConcatListOfFrames
              ++linearBlockIdx
              )
         {
+            T_Filter filter = filterWindowDomain;
```

Comment:
@ax3l  
- openmp `private(var)` means that the variable is not initialized per thread.
- openmp  `firstprivate(var)` can used but it has some [restrictions](https://www.ualberta.ca/CNS/RESEARCH/NumStatsServers/Intel/Documentation/ssadiag_docs/index.htm#pt_reference/references/sc_omp_private_uninit.htm)